src/Liip/RMT/Action/UpdateVersionClassAction.php

```diff
@@ -29,6 +29,7 @@
  * }
  *
  * @author David Buchmann <mail@davidbu.ch>
+ * @deprecated Please use UpdateVersionFilesAction instead
```

Comment:
Shouldn't it be instead:
```suggestion
 * @deprecated Please use FilesUpdateAction instead
```

Reply:
Good catch @seyfahni !